modules/mixed-arch-modify-machine-set.adoc

```diff
@@ -0,0 +1,76 @@
+//Module included in the following assembly
+//
+//post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="mixed-arch-modify-machine-set_{context}"]
+
+= Adding a machine set to your cluster using the `arm64` bootimage  
+
+To add `arm64` worker nodes to your heterogeneous cluster, you must create an Azure machine set that uses the `arm64` bootimage. To create your own custom machine set on Azure, see "Creating a machine set on Azure". 
+
+.Prerequisites 
+
+* Installed the OpenShift CLI (`oc`). 
+
+.Procedure
+. View the machine sets in the cluster:
++
+[source,terminal]
+----
+oc get machineset -o yaml my-machine-set.yaml
+----
+. Create a machine set and modify the `resourceID` and `vmside` parameters. This machine set will control the `arm64` worker nodes in your cluster.
++
+[source,terminal]
+----
+oc create -f my-machine-set.yaml
+----
+.Sample YAML machine set with `arm64` bootimage 
++
+[source,yaml]
+----
+    providerSpec:
+        value:
+          apiVersion: azureproviderconfig.openshift.io/v1beta1
+          credentialsSecret:
+            name: azure-cloud-credentials
+            namespace: openshift-machine-api
+          image:
+            resourceID: /resourceGroups/${RESOURCE_GROUP}/providers/Microsoft.Compute/galleries/${GALLERY_NAME}/images/rhcos-arm64/versions/1.0.0 <1>
+          kind: AzureMachineProviderSpec 
+          location: <region> 
+          managedIdentity: <infrastructure_id>-identity 
+          metadata:
+            creationTimestamp: null
+          natRule: null
+          osDisk:
+            diskSizeGB: 128
+            managedDisk:
+              storageAccountType: Premium_LRS
+            osType: Linux
+          publicIP: false
+          resourceGroup: <infrastructure_id>-rg 
+          subnet: <infrastructure_id>-<role>-subnet   
+          userDataSecret:
+            name: worker-user-data 
+          vmSize: Standard_D4s_v3 <2>
```

Comment:
This ex VMsize should work right?

Reply:
`Standard_D4ps_v5` or `D8ps` are typical arm64 instances. the above is a default x86 instance